clang/cmake/caches/Standalone_libc_baremetal.cache

```diff
@@ -0,0 +1,43 @@
+# Expects target triple to be passed as `ARCH_TRIPLE`
```

Comment:
This seems to be a bit odd being a clang CMake cache. We should probably move it under the `libc/` subdirectory.